### ISSUE TITLE
ERROR installing on Antergos: using `aura`

### ISSUE THREAD

**Issue by aniketd** [Eloston/ungoogled-chromium#186]
```shell
Cloning into 'ungoogled-chromium'...
done.
==> Starting pkgver()...
==> Starting build()...
==> Building...this will take a while...
2017-02-06 14:43:27,816 - INFO: Initialized default console logging handler
2017-02-06 14:43:27,816 - INFO: Using builder LinuxStaticBuilder
2017-02-06 14:43:27,817 - INFO: Directory build does not exist. Creating...
2017-02-06 14:43:27,818 - INFO: Directory build/sandbox does not exist. Creating...
2017-02-06 14:43:27,818 - INFO: Directory build/downloads does not exist. Creating...
2017-02-06 14:43:27,818 - INFO: Directory build/path_overrides does not exist. Creating...
2017-02-06 14:43:27,819 - INFO: Setting up environment overrides...
2017-02-06 14:43:27,819 - INFO: Checking Python 2 command...
2017-02-06 14:43:27,819 - INFO: No Python 2 command specified; testing with 'python'
2017-02-06 14:43:27,856 - ERROR: Unsupported Python version '3.6.0'
==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by Eloston** [comment]
Whoops. Will be fixed with #125.

**Comment by Eloston** [comment]
Closing since `buildlib` has been deprecated a while ago.
